### DISCUSSION TITLE
blog/self-host-your-nuxt-app-with-coolify

### DISCUSSION THREAD

**Discussion by giscus** [Mokkapps/mokkapps-de-discussions#18] [Announcements]
# blog/self-host-your-nuxt-app-with-coolify

I share my experiences of how I self-hosted my Nuxt apps with Coolify on Hetzner servers.

https://mokkapps.de/blog/self-host-your-nuxt-app-with-coolify

**Comment by mirsella** [comment]
hello, thanks for the article. im trying to put a test nuxtjs app to /test instead of /.
the request to / is good, but then all nuxtjs assets in /test/_nuxt don't work !
```
Loading module from “http://localhost/_nuxt/BHf242N9.js” was blocked because of a disallowed MIME type (“text/plain”)
```
i would love any help ! thank you !

**Reply by Mokkapps** [reply]
Hi, unfortunately it's nearly impossible to help you without a reproduction of your code. 

Can you confirm that it's not a bug in your app? If you build your production bundle locally and preview the output, do you encounter the same error?

**Reply by mirsella** [reply]
hey, thanks a lot for the answer.

i found the problem ! it was simply the "strip prefix" option. 
sadly i was testing lot of things and didn't manage to isolate this option.

i didn't give more info because thought it was a coolify issue and not nuxt as i was testing with a newly default created nuxt app, and still not working. i coudn't find any resource online about hosting with a different sub path in coolify, i was strating to doubt coolify was supporting it.

have a good day !
